app/minutes/controllers.py

```diff
@@ -14,6 +14,38 @@
 from app.minutes.minutes_response import Response
 from app.users.models import Users
 
```

Comment:
Please add Doxygen comments to your function (My function is also missing them, oops!). You can find an example of Doxygen comments in [Charge controllers](https://github.com/ritstudentgovernment/chargeflask/blob/master/app/charges/controllers.py#L48). I think there is a VSCode extension for Doxygen, I found [this one](https://marketplace.visualstudio.com/items?itemName=cschlosser.doxdocgen) that is very similar.

Reply:
Thanks for the comments!